### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by amyblais**
Please help provide any missing details for these new config settings.

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2323,6 +2323,14 @@ The issuer URL for the Identity Provider you use for SAML requests.
 | This feature's ``config.json`` setting is ``"IdpDescriptorUrl": ""`` with string input.                                                                              |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Identity Provider Metadata URL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The metadata URL for the Identity Provider you use for SAML requests.
```
Comment (sbishel) [inline source/administration/config-settings.rst]:
```suggestion
The URL where Mattermost sends a request to obtain setup metadata from the provider.
```

**Comment by amyblais** [comment]
@sbishel Based on the comment here https://github.com/mattermost/docs/pull/3314#issuecomment-585197523, SamlSettings.EnableAdminAttribute and SamlSettings.AdminAttribute were also added to LdapSettings, do I document these here too?

**Comment by sbishel** [comment]
@amyblais - 
Sorry, I got confused, since we just documented those here as well.
https://github.com/mattermost/docs/pull/3359

Yes, there are two sets missing, 
 - LdapSettings { EnableAdminFilter, AdminFilter}
 - SamlSettings { EnableAdminAttribute, AdminAttribute}

Let me know if you need additional info.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2739,6 +2746,17 @@ Enable Plugins
 | This feature's ``config.json`` setting is ``"Enable": true`` with options ``true`` and ``false``.                                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Automatic Prepackaged Plugins
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: The server will detect previously enabled plugins on the server and will automatically install them.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]:
```suggestion
**True**: Any pre-packaged plugins enabled in the configuration will be installed or upgraded automatically. If a newer version is already installed, no changes are made.
```

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -2739,6 +2746,17 @@ Enable Plugins
...
+
+**False**: The server will not detect previously enabled plugins on the server.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]:
```suggestion
**False**: Pre-packaged plugins are not installed or upgraded automatically but may be installed manually from the Plugin Marketplace, even when offline.
```

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -2750,6 +2768,17 @@ Enable Marketplace
 | This feature's ``config.json`` setting is ``"EnableMarketplace": true`` with options ``true`` and ``false``.                                                         |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Remote Marketplace
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: Marketplace fetches latest plugins from the configured Marketplace URL.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]:
```suggestion
**True**: The server will attempt to connect to the configured Plugin Marketplace to show the latest plugins. If the connection fails, the Plugin Marketplace shows only pre-packaged and already installed plugins alongside a connection error.
```

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -2750,6 +2768,17 @@ Enable Marketplace
...
+
+**False**: Marketplace does not fetch latest plugins from the configured Marketplace URL.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]:
```suggestion
**False**: The server will not attempt to connect to a remote marketplace, instead showing only pre-packaged and already installed plugins. Use this setting if your server cannot connect to the internet.
```

**Pull request merged**